docs/getting-started/deploy-singlenode-cluster.md

```diff
@@ -0,0 +1,103 @@
+
+
+| Method | Required Installation Files | Other Requirements |
+| --- | --- | --- |
+| [ISO](../install/index) | ISO | N/A |
+| [USB](../install/usb-install) | ISO | USB flash drive; utility such as [balenaEtcher](https://etcher.balena.io/) or the Linux [dd command](https://man7.org/linux/man-pages/man1/dd.1.html) |
+| [PXE](../install/pxe-boot-install) | ISO, Linux kernel image (vmlinuz), initrd, SquashFS image | Directory on the HTTP server (for serving boot files); iPXE boot scripts (for automatic installation); DHCP server configuration |
+
+## 3. Prepare the cluster configuration requirements. 
+
+- Cluster token: ASCII string that nodes use when joining the cluster 
+- Fixed IP address for each node: May be assigned statically or using DHCP (host reservation) 
+- Fixed virtual IP address (VIP) to be used as the cluster management address: VIP that you connect to when performing administration tasks after the cluster is deployed 
+- Addresses of DNS servers, NTP servers, and the proxy server (if necessary) 
+
+## 4. Deploy the cluster node. 
+
+Deployment involves installing the operating system and other components on the host, and then rebooting once installation is completed. Deploying the node creates the cluster, and the node is assigned the management role by default. 
+    
+During installation, you must configure node settings, define the cluster management address (VIP) and the cluster token, and specify other information. If necessary, you can configure more settings using a [Harvester configuration](../install/harvester-configuration/) file. 
+
+Once installation is completed, the node restarts and then the Harvester console appears. The console displays information about the cluster (management URL and status) and the node (hostname, IP address, and status). After the cluster is initialized and all services start running, the cluster status changes to **Ready**. 
+
+## 5. Configure a strong password for the default `admin` user on the Harvester UI. 
+
+Once the cluster status changes to **Ready**, you can access the [Harvester UI](../authentication) using the management URL displayed on the console. 
+
+## 6. Create a new default StorageClass.
+
+Creating a new [StorageClass](../advanced/storageclass/#creating-a-storageclass) is necessary because the default StorageClass `harvester-longhorn` has a replica count value of **3** for high availability. If you use `harvester-longhorn` to create volumes, Longhorn is unable to create the configured number of replicas and volumes are marked as *Degraded* on the Longhorn UI.
+
+When configuring the new StorageClass, set the **Number of Replicas** parameter to **1** so that only one replica is created for each Longhorn volume. 
+
+## 7. Create a custom cluster network (optional) and a VM network (required).
```

Comment:
Suggest move step 7~10 to `deploy-ha-cluster` and refer to them from this file

Reply:
The `VM network` is not `required`, user can deploy VM on management-network, for POC or trial or test, that's fine.

Reply:
> Suggest move step 7~10 to deploy-ha-cluster and refer to them from this file

The procedure is designed to be self-contained. All crucial information, particularly required steps, must be in the same file to limit interruptions to the reader's flow.

> The VM network is not required, user can deploy VM on management-network, for POC or trial or test, that's fine.

Thank you for the information. Step 7 is now marked optional. I also added a sentence about how using the management network is acceptable.